src/manifest/pfm.rs

```diff
@@ -82,14 +91,60 @@ impl<F, P> Manifest for Pfm<'_, F, P> {
     }
 }
 
-impl<'pfm, F, P> Pfm<'pfm, F, P> {
+impl<'f, F: 'f + Flash, P> Parse<'f, F, P> for Pfm {
+    type Parsed = ParsedPfm<'f, F, P>;
+
+    fn parse(
+        container: Container<'f, Self, F, P>,
+    ) -> Result<Self::Parsed, Error> {
+        Ok(ParsedPfm::new(container))
+    }
+
+    fn copy_to<F2: Flash>(
+        manifest: &Self::Parsed,
+        dest: &mut F2,
+    ) -> Result<(), Error> {
+        let src = manifest.container.flash();
+        let len = src.size()? as usize;
+        let mut bytes_left = len;
+
+        let mut buf = [0; 32];
```

Comment:
nit: `[0u8; 32]` to be more obvious this is a byte buffer.

nit2: 32 bytes is pretty small for flash - generally a "page" in flash is 256 bytes (I see your flash trait specifically says that `program` is expected to optimize the actual writing to flash.  Given the differences between sectors (typ. 4K) and pages (256B), you might want to consider more documentation on `program` regarding the expected interactions between erasing and writing and whether or not any preservation of pre-existing content is expected.  I don't have any specific recommendation for this PR regarding "nit2", but rather to consider the behavior of the `Flash` trait).